src/pages/round/[...number].tsx

```diff
@@ -41,7 +41,7 @@ export default function Round() {
   const number = router.query.number || "";
   const { voteActive, setVoteActive, requestedRound, display, setDisplay } = useGlobalContext();
   const bribeData = trpc.bribes.list.useQuery(
-    { round: requestedRound },
+    { round: parseInt(number[0] as string) },
```

Comment:
I'm assuming this has to be this way, but damn ... i'm going to merge and look at this closer later.